dart/lib/src/scope.dart

```diff
@@ -142,6 +142,59 @@ class Scope {
   /// Removes an extra from the Scope
   void removeExtra(String key) => _extra.remove(key);
 
+  SentryEvent applyToEvent(SentryEvent event, dynamic hint) {
+    if (event.transaction == null) {
+      event = event.copyWith(transaction: transaction);
+    }
+
+    // Merge the user context.
```

Comment:
I dont think the wording is `merge` right, its either one or the other

Reply:
✅